las.go

```diff
@@ -97,6 +58,41 @@ func (l *LasType) Data() [][]string {
 	return chunk(sBs, len(hds))
 }
 
+// DataStruct just like Data but returns output in specified struct format
+func (l *LasType) DataStruct(opt *DataOptions) []interface{} {
+
+	hds, err := l.Header()
+	if err != nil {
+		panic("No data in file")
+	}
+	sB := pattern("~A(?:\\w*\\s*)*\n").Split(l.content, 2)[1]
+	sBs := pattern("\\s+").Split(strings.TrimSpace(sB), -1)
+
+	var (
+		o     *DataOptions
+		store [][]string
+	)
+
+	if opt != nil {
+		o = opt
+	}
+
+	store = chunk(sBs, len(hds))
+
```

Comment:
@propersam can we replace this block with `store := *l.Data()`

Reply:
hmm...that's possible ...i also see repitition... lemme do that asap